src/utils/importFuncGenerator.ts

```diff
@@ -148,113 +72,30 @@ export function createImport(
   if (warplibFunc !== undefined) {
     return createFuncImport(...warplibFunc);
   }
-
-  switch (encodePath([path, name])) {
-    case encodePath(ALLOC):
-      return createFuncImport();
-    case encodePath(BITWISE_BUILTIN):
-      return createStructImport();
-    case encodePath(FINALIZE_KECCAK):
-      return createFuncImport('range_check_ptr', 'bitwise_ptr');
-    case encodePath(DEFAULT_DICT_NEW):
-      return createFuncImport();
-    case encodePath(DEFAULT_DICT_FINALIZE):
-      return createFuncImport('range_check_ptr');
-    case encodePath(DICT_READ):
-    case encodePath(DICT_WRITE):
-      return createFuncImport('dict_ptr');
-    case encodePath(DICT_ACCESS):
-      return createStructImport();
-    case encodePath(U8_FROM_FELT):
-    case encodePath(U16_FROM_FELT):
-    case encodePath(U24_FROM_FELT):
-    case encodePath(U32_FROM_FELT):
-    case encodePath(U40_FROM_FELT):
-    case encodePath(U48_FROM_FELT):
-    case encodePath(U56_FROM_FELT):
-    case encodePath(U64_FROM_FELT):
-    case encodePath(U72_FROM_FELT):
-    case encodePath(U80_FROM_FELT):
-    case encodePath(U88_FROM_FELT):
-    case encodePath(U96_FROM_FELT):
-    case encodePath(U104_FROM_FELT):
-    case encodePath(U112_FROM_FELT):
-    case encodePath(U120_FROM_FELT):
-    case encodePath(U128_FROM_FELT):
-    case encodePath(U136_FROM_FELT):
-    case encodePath(U144_FROM_FELT):
-    case encodePath(U152_FROM_FELT):
-    case encodePath(U160_FROM_FELT):
-    case encodePath(U168_FROM_FELT):
-    case encodePath(U176_FROM_FELT):
-    case encodePath(U184_FROM_FELT):
-    case encodePath(U192_FROM_FELT):
-    case encodePath(U200_FROM_FELT):
-    case encodePath(U208_FROM_FELT):
-    case encodePath(U216_FROM_FELT):
-    case encodePath(U224_FROM_FELT):
-    case encodePath(U232_FROM_FELT):
-    case encodePath(U240_FROM_FELT):
-    case encodePath(U248_FROM_FELT):
```

Comment:
Won't the lack of those break uintN PR functionality? Since we've got rid of switch we could also do it smarter now